src/hubbleds/story.py

```diff
@@ -185,6 +188,32 @@ def _set_theme(self):
         v.theme.themes.light.warning = 'colors.deepOrange.accent4'
         #Alt Palette 1:  Y:FFBE0B, O:FB5607, Pi:FF006E, Pu:8338EC, Bl:3A86FF, LiBl:619EFF
 
+    def _setup_all_data(self):
+        # Load in the overall data
+        all_json = self._request_session.get(f"{API_URL}/{HUBBLE_ROUTE_PATH}/all-data?before={self.started}").json()
+        all_measurements = all_json["measurements"]
+        for measurement in all_measurements:
+            measurement.update({"galaxy_id": measurement["galaxy"]["id"]})
+            measurement.pop("galaxy")
+            measurement.pop("student")
```

Comment:
Because this code block moved, it does not get updated when main is merged in (at least for me). 
```suggestion
            measurement.update({"galaxy_id": measurement["galaxy_id"]})
            measurement.pop("galaxy", None)
            measurement.pop("student", None)
```